x-pack/test/fleet_api_integration/apis/epm/install_remove_assets.ts

```diff
@@ -279,6 +279,17 @@ export default function (providerContext: FtrProviderContext) {
           resOsquerySavedQuery = err;
         }
         expect(resOsquerySavedQuery.response.data.statusCode).equal(404);
+        let securityAiPrompt;
+        try {
+          securityAiPrompt = await kibanaServer.savedObjects.get({
+            type: 'security-ai-prompt',
+            id: 'sample_security_ai_prompt',
```

Comment:
Is it possible to have conflicts if multiple packages have prompts with the same id?

I think that for other assets we prefix their ids with the name of the package to avoid these issues.

Reply:
The ids will be UUID autogenerated by saved object client, so we will not have this issue. this id was just for tests following other examples